validator/include/platform_types.h

```diff
@@ -29,11 +29,22 @@
 
 #include <cstdint>
 
+#ifdef RV64_VALIDATOR
+#define ADDRESS_T_MAX UINT64_MAX
```

Comment:
Can we have an XLEN variable with the value 32/64 and eliminate these `ifdef`s?

Reply:
Wouldn't that just replace it with a different ifdef?  (Not that changing the macro name wouldn't be a good change.)

Reply:
Now that I'm thinking about the cpp technique you'd use to do this, this looks like the simpler way